roles/config-vnc-server/tasks/prereq.yml

```diff
@@ -1,12 +1,16 @@
 ---
 
+- name: 'Determine required python firewall package'
+  set_fact: python_firewall_package="python3-firewall"
+  when: ansible_python_version is match("3.*")
```

Comment:
We typically structure the `when` clause on multiple lines for better readability + easier to add additional checks in the future. e.g.:
```
when:
- ansible_python_version is match("3.*")
```

Reply:
@oybed No problem. The only question I have is whether we actually want to include the `ansible_python_interpreter` in the sample inventory.  As of now, leaving this alone (not specifying) lets the interpreter just run with whatever the systems default python is. At this point, I would think that's the desired behavior. Fedora 28 seems to make the mistake of removing some of the old python 2.X packages, but not specifically specifying the system python to be 3.X.

Would an example like this be better served by a README that contains a warning to include this if you plan on utilizing python 3.X (or specifically if you want to run Fedora 28)? I see this python firewall package in a few other roles as well, so probably whatever we decide here will get used elsewhere.

Reply:
The sample inventory is truly that - i.e.: an example of how to make it run with some specific versions. I think we need to come up with a solution that applies to the other places, but for the `bastion` sample, I'd say just include it with some comments around why it's needed. I.e.:

Update the https://github.com/redhat-cop/infra-ansible/blob/master/inventory/bastion/group_vars/all.yml#L45 to use the `F28` image, and update  https://github.com/redhat-cop/infra-ansible/blob/master/inventory/bastion/group_vars/bastion.yml with the `ansible_python_interpreter` + a comment about why it's needed in combination with the F28 image / python3. 

Feel free to update the README for this particular role, but I think we need to dig in a bit more for the rest as we don't really have a good place to capture it "globally" (+ I'm thinking some other changes/fixes will come out for this soon anyway).